patches/chrome-installer-linux-BUILD.gn.patch

```diff
@@ -43,7 +43,21 @@ index 4ac4a301c73841290fac70097e379b786108f831..054b8f885497e760aa06ee9762e6acf1
      ]
    }
  
-@@ -354,6 +359,12 @@ group("installer_deps") {
+@@ -310,6 +315,13 @@ copy("theme_files") {
+       "$branding_dir_100/product_logo_32_dev.png",
+     ]
+   }
++  else {
++    sources += [
++      "$branding_dir/product_logo_128_beta.png",
++      "$branding_dir/product_logo_128_dev.png",
++      "$branding_dir/product_logo_128_development.png",
```

Comment:
how about handing nightly image also?

Reply:
I think we can avoid this patching and creating same copy target to our gn(`ex, brave/BUILD.gn`) and adding it proper deps list like `create_dist` target.